server/server.js

```diff
@@ -129,6 +165,83 @@ app.post('/api/sign-in', async (req, res, next) => {
   }
 });
 
+// User can add to shopping cart
+app.post(
+  '/api/orderItems/:orderItemId',
+  authMiddleware,
+  async (req, res, next) => {
+    try {
+      const { orderId, productId, quantity } = req.body;
```

Comment:
The orderId could come here, or more likely from `req.params.orderId`.

Reply:
Is this what you meant?
const orderId = req.params.orderId; 
const { productId, quantity } = req.body;  

or is it only const orderId = req.params.orderId; ?